### ISSUE TITLE
Folder structure as a tree

### ISSUE THREAD

**Issue by OmlineEditor** [Kunzisoft/KeePassDX#439]
in the main window, I now see folders as folders without their contents. I click on the folder and see the contents of the folder. I don't want to click on a folder and see its contents I want to see all the contents of the folder including the folded folders in the form of a tree.

this is more convenient. you can see everything at once.

**Comment by J-Jamet** [comment]
I agree, it would be much better but requires many modifications. I'll think about it once the application is in production.

**Comment by Wilker-a** [comment]
it would be useful if it was implemented and mixed with the current model of changing screens.

- the `›` changes direction and expand the items
- while clicking the folder still brings you to a screen of its own

easy to understand, direct, and not much of a need for changing default settings unless people wants to only have either :3
